mne/io/open.py

```diff
@@ -258,8 +273,10 @@ def _show_tree(fid, tree, indent, level, read_limit, max_str):
                 out[-1] = out[-1].replace('\n', u'¶')
                 counter = 0
                 good = True
-
+    if tag_id is not None and not tag_found:
```

Comment:
this can also just be `if not tag_found` because the only time that happens is when `tag_id is not None`